decidim-conferences/app/cells/decidim/conferences/conference_speaker/show.erb

```diff
@@ -1,87 +1,73 @@
-<div class="column conference-speaker js-conference">
-  <div class="speaker-img-wrapper">
-    <div class="speaker-img">
-      <%= image_tag avatar_path, alt: "speaker-image" %>
-    </div>
+<div class="conference__speaker__item" data-conference-speaker data-dialog-open="<%= model.id %>">
+  <div class="conference__speaker__item-image">
+    <%= image_tag avatar_path, alt: nickname || "" %>
   </div>
-  <div class="speaker-data">
-    <div class="data-name"><%= name %></div>
-    <% if position.presence %>
-      <div class="data-role">
-        <%= position %>
-      </div>
-    <% end %>
-    <% if affiliation.presence %>
-      <div class="data-extra">
-        <%= affiliation %>
-      </div>
-    <% end %>
-    <div class="data-extra">
-      <% if nickname.present? %>
-        <%= link_to nickname, profile_path, class: "card-link" %>
-      <% else %>
-        <div class="author__nickname">&nbsp;</div>
+  <div>
+    <h2 class="h5 conference__speaker__item-name"><%= name %></h2>
+    <div class="conference__speaker__item-text">
+      <% if position.presence %>
+        <span><%= position %></span>
+      <% end %>
+      <% if affiliation.presence %>
+        <span><%= affiliation %></span>
       <% end %>
-      <div><u><%= t(".more_info") %></u></div>
     </div>
   </div>
-  <div class="speaker-bio js-bio">
-    <div class="row">
-      <div class="column medium-6">
-        <div class="speaker-img-wrapper">
-          <div class="speaker-img">
-            <%= image_tag avatar_path, alt: "speaker-image" %>
-          </div>
-        </div>
-        <div class="speaker-data">
-          <div class="data-name"><%= name %></div>
+</div>
+
+<%= decidim_modal id: model.id, class: "conference__speaker__modal" do %>
+  <div class="conference__speaker__modal-title">
+    <%= icon "user-voice-line", class: "w-6 h-6 text-gray fill-current" %>
+    <span><%= t("decidim.conferences.conference_speakers.index.speakers", count: 1) %></span>
```

Comment:
We don't have currently the correct heading levels marked in the modals as required by WCAG.

Relevant criteria:
- [1.3.1 Info and Relationships](https://www.w3.org/WAI/WCAG21/quickref/#qr-content-structure-separation-programmatic)
- [2.4.6 Headings and Labels](https://www.w3.org/WAI/WCAG21/quickref/#qr-navigation-mechanisms-descriptive)
- [2.4.10 Section Headings](https://www.w3.org/WAI/WCAG21/quickref/#qr-navigation-mechanisms-headings)